packages/app/app/layout.tsx

```diff
@@ -7,6 +7,12 @@ export const metadata = {
   title: "Stackly | Stack crypto over time.",
   description:
     "Stackly is a simple, non-custodial tool that uses the CoW protocol to place recurring swaps based on DCA..",
+  viewport: {
+    width: "device-width",
+    initialScale: 1,
+    maximumScale: 1,
+    userScalable: 0,
+  },
```

Comment:
why?

Reply:
We need this to prevent zoom in on mobile when clicking on inputs on iOS

Reply:
I think that only happens on `iOS` not a big deal. Don't know if  has implications in other things, but let's keep it for now.